src/styles/blocks/header.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
```

Comment:
It's better to put the whole nav block and all the children element in a seperate file like navigation.scss inside blocks for better structure.

Reply:
yeah, I agree with @CharAnagn about this point